src/mudlet.pro

```diff
@@ -50,7 +52,7 @@ exists("../3rdparty/edbee-lib/edbee-lib/edbee-lib.pri") {
 
 # Set the current Mudlet Version, unfortunately the Qt documentation suggests
 # that only a #.#.# form without any other alphanumberic suffixes is required:
-VERSION = 3.5.0
+VERSION = 3.5.7
```

Comment:
You may want to reset this value to 3.5.0 again

Reply:
Fixed